datafusion/proto/proto/datafusion.proto

```diff
@@ -1206,13 +1206,28 @@ message PartitionColumn {
 message FileTypeWriterOptions {
   oneof FileType {
     JsonWriterOptions json_options = 1;
+    ParquetWriterOptions parquet_options = 2;
   }
 }
 
 message JsonWriterOptions {
   CompressionTypeVariant compression = 1;
 }
 
+message ParquetWriterOptions {
+    WriterProperties writer_properties = 1;
+}
+
+message WriterProperties {
+  int32 data_page_size_limit = 1;
```

Comment:
given the rust fields are usize, is there any reason to use in32 in the protobuf encoding? 

Maybe it would make more sense to use `i64` or `u64` here instead 🤔 

https://protobuf.dev/programming-guides/proto2/#scalar

Reply:
Good point. I have updated to u32.

Reply:
Actually, I misread / misunderstood your comment. I changed to u32 on the basis that these should not need to support negative numbers, but I may need to research this more to make sure that is the case.

Reply:
Never mind, in Rust they are usize, so can't be negative.

Reply:
Now changed to u64 to better match usize.